apps/meteor/app/livechat/server/api/lib/livechat.ts

```diff
@@ -126,7 +126,7 @@ export function getRoom({
 	return LivechatTyped.getRoom(guest, message, roomInfo, agent, extraParams);
 }
 
-export async function findAgent(agentId: string): Promise<void | { hiddenInfo: true } | ILivechatAgent> {
+export async function findAgent(agentId?: string): Promise<void | { hiddenInfo: true } | ILivechatAgent> {
```

Comment:
is this really correct? why you are trying to find the agent if you dont have the id?